### ISSUE TITLE
Accessibility: Localized Control Type property is wrong for the renderer chooser dropdown button

### ISSUE THREAD

**Issue by bojanrajkovic** [Microsoft/workbooks#134]
### Steps to Reproduce

1. Launch Workbooks or inspect an application
2. Run `new object { hello = "world" }` or select a view in the inspector
3. Check the Localized Control Type property of the renderer chooser dropdown (`ToString ()`)

### Expected Behavior

Should be identified as a dropdown/menu

### Actual Behavior

Is identified as text, as per screenshot.
![image](https://user-images.githubusercontent.com/110747/33854475-d99ca488-de8f-11e7-9548-9514e04b2114.png)

### Environment

* OS: Windows 10, version 1067, build 14393.607.
* Visual Studio 2017 RC Enterprise
* Xamarin for VS 4.3.0.550
* Xamarin Inspector/Workbooks 1.0

This is a port of https://bugzilla.xamarin.com/show_bug.cgi?id=51824.
